dist/image-sequencer-ui.js

```diff
@@ -338,11 +339,16 @@ function DefaultHtmlStepUi(_sequencer, options) {
   function onDraw(step) {
     $(step.ui.querySelector(".load")).show();
     $(step.ui.querySelector("img")).hide();
+    if( $(step.ui.querySelector(".toggleIcon")).attr("class").includes("fa-caret-down") )
```

Comment:
You can use the `$.hasClass()` method which returns a boolean.

Reply:
Good call @HarshKhandeparkar :+1: